src/networks.ts

```diff
@@ -108,6 +109,7 @@ export const explorers = [
   "https://holesky.fraxscan.com/",
   "https://sepolia-explorer.metisdevops.link/",
   "https://sepolia.explorer.mode.network/",
+  "https://evm-rpc-arctic-1.sei-apis.com",
```

Comment:
This is an RPC; it requires a block explorer link here...

Reply:
ok, I have fixed this